profiler/CMakeLists.txt

```diff
@@ -45,13 +48,28 @@ if(THEROCK_ENABLE_ROCPROFV3)
   ##############################################################################
   # rocprofiler-sdk
   ##############################################################################
+  # Detect available Default Python3 version on the system or
+  # use explicitly provided Python3 versions
+  therock_detect_python_versions(_detected_python_executables _detected_python_versions)
+
+  # Use detected versions or fall back to default if none found
+  if(NOT _detected_python_versions)
+    message(WARNING "No Python versions detected for rocprofiler-sdk")
+    set(_detected_python_versions_str "")
+  else()
+    message(STATUS "Detected Python versions for rocprofiler-sdk python bindings: ${_detected_python_versions}")
+    message(STATUS "Python executables: ${_detected_python_executables}")
+    list(JOIN _detected_python_versions "\;" _detected_python_versions_str)
```

Comment:
This is a weird encoding. Can you fix rocprofiler-sdk to have maybe two options vs a weird interleaving?

(can be a followup if this works as-is)